src/connections/DelayedJob.ts

```diff
@@ -14,56 +19,76 @@ type DelayedJobCombinedConnection = {
   delayedJobsHasFailed: boolean;
   bulkUpdateJobsIsLoading: boolean;
   bulkUpdateJobsHasFailed: boolean;
-  updatedJobsResponse?: DelayedJobDto[];
 };
 
-const delayedJobsSelector = (store: ApiDataStore) => store.delayedJobs;
+const delayedJobsSelector = (store: ApiDataStore) =>
+  Object.values(store.delayedJobs ?? {})
+    .map(resource => resource.attributes)
+    .filter(({ isAcknowledged }) => !isAcknowledged);
 
 const combinedSelector = createSelector(
   [delayedJobsSelector, bulkUpdateJobsIsFetching, bulkUpdateJobsFetchFailed],
   (delayedJobs, bulkUpdateJobsIsLoading, bulkUpdateJobsFailure) => ({
-    delayedJobs: Object.values(delayedJobs ?? {}).map(resource => resource.attributes),
+    delayedJobs,
     delayedJobsIsLoading: delayedJobs == null && !bulkUpdateJobsFailure,
     delayedJobsHasFailed: Boolean(bulkUpdateJobsFailure),
     bulkUpdateJobsIsLoading,
-    bulkUpdateJobsHasFailed: bulkUpdateJobsFailure != null,
-    updatedJobsResponse: Object.values(delayedJobs ?? {}).map(resource => resource.attributes as DelayedJobDto)
+    bulkUpdateJobsHasFailed: bulkUpdateJobsFailure != null
   })
 );
 
-const combinedLoad = (connection: DelayedJobCombinedConnection) => {
-  if (!combinedIsLoaded(connection)) {
-    listDelayedJobs();
-  }
-};
-
-const combinedIsLoaded = ({
-  delayedJobs,
-  delayedJobsHasFailed,
-  bulkUpdateJobsIsLoading,
-  bulkUpdateJobsHasFailed
-}: DelayedJobCombinedConnection) =>
-  (delayedJobs != null || delayedJobsHasFailed) && !bulkUpdateJobsIsLoading && !bulkUpdateJobsHasFailed;
-
 const delayedJobsCombinedConnection: Connection<DelayedJobCombinedConnection> = {
-  load: combinedLoad,
-  isLoaded: combinedIsLoaded,
   selector: combinedSelector
 };
 
+export const useJobProgress = () => useSelector<AppStore, JobsDataStore>(({ jobs }) => jobs);
```

Comment:
This is literally all that's needed to get immediate updates when a section of the redux store changes.